app/controllers/settings/exports/blocked_accounts_controller.rb

```diff
@@ -2,7 +2,7 @@
 
 module Settings
   module Exports
-    class BlockedAccountsController < ApplicationController
+    class BlockedAccountsController < BaseController
```

Comment:
ok, if this inherits from `Settings::BaseController`, doesn't that still `require_functional!`?

Reply:
`ExportsControllerConcern` adds `skip_before_action :require_functional!`

Reply:
Right! Sorry about that.